Scoop/Controller/Ride Posting/PostRideSummaryViewController.swift

```diff
@@ -98,7 +98,7 @@ class PostRideSummaryViewController: UIViewController {
         
         let organizerLabel = UILabel()
         organizerLabel.font = .systemFont(ofSize: 12)
-        organizerLabel.text = "Organizer: @\(ride.organizer.username)" // TODO: We need to support Usernames during onboarding
+        organizerLabel.text = "Organizer: @\(ride.creator.netid)" // TODO: We need to support Usernames during onboarding
```

Comment:
Are usernames now supported during Onboarding?

Reply:
Spoke with Kate earlier today, the username param in backend just takes a Cornell email now